app/models/setting.rb

```diff
@@ -58,4 +58,12 @@ def self.default_start_date
   def self.fb_reachable_period
     (ENV["FB_REACHABLE_DAY"] || 1).to_i
   end
+
+  def self.pilot_provinces
+    ENV["PILOT_PROVINCES"].split(",")
```

Comment:
@radinreth prevent in case this environment variable didn't present(nil)

Reply:
it return [] if ENV["PILOT_PROVINCE_CODES"] is not set